### PR TITLE
Support officialy PHP 8.0 in CI

### PR DESCRIPTION
**Pull request by deguif**

### CODE REVIEW

**Comment by deguif** [comment]
@ruflin Can you review this one?
I would like to get this one merged before merging the PR preparing the new release: #1871

**Review by ruflin — APPROVED** [review]
LGTM. I think I lack some background on why it is need but please don't block on that to get it in.

**Pull request merged**
